WebDevStudios/wd_s
# Reorganize the inc/ directory

Closes #764 

### DESCRIPTION

Reorganizes the `inc/` directory to improve code organization and maintainability. Each function is housed in its own file and organized in directories that describe the function/purpose of the code (e.g. `inc/template-tags/demo-function.php`).

_Note: The only real code changes are in `functions.php` and the doc comments._

### SCREENSHOTS

![inc-dir_03](https://user-images.githubusercontent.com/1083785/137370669-e207fd45-d855-4b74-87f9-193d52f12906.jpg)

### OTHER


### STEPS TO VERIFY

This update does not add/change any functionality, so just make sure nothing is broken in the theme.

### DOCUMENTATION

Will this pull request require updating the wd_s [wiki](https://github.com/WebDevStudios/wd_s/wiki)?
Yes
